src/iojunk.jl

```diff
@@ -3,7 +3,7 @@ function write{T}(w::WritableFile, a::Array{T})
 	# If this is not provided, Base.IO write methods will write
```

Comment:
the corresponding method in base should be more efficient than these now (due to dispatching to `unsafe_write` more uniformly)